src/test/resources/mapping-rules.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "kafkaTopic": "building_{building}_room_{room}",
+    "mqttTopic": "building/{building}/room/{room}/#"
```

Comment:
Personally would swap the positions of kafkaTopic and mqttTopic like we did in the last PR